src/components/button-group/button-group.scss

```diff
@@ -146,6 +146,7 @@
 
     &.disabled {
         @include shared_input-select-picker.looks-disabled;
+        pointer-events: none;
```

Comment:
same as [`03a0c96` (#2771)](https://github.com/Lundalogik/lime-elements/pull/2771/commits/03a0c964e4ccaac2d6489f4dfd1f22ed5595d6ef#r1481182430)

Reply:
I don't understand this comment… 😅

Reply:
Ah, you meant to link to the comment on `chip-set.scss` I assume? Then I understand 👍